packages/babel-plugin-transform-react-inline-elements/test/fixtures/regressions/6276/actual.js

```diff
@@ -0,0 +1,3 @@
+var test = <T default="
+    some string
```

Comment:
@nicolo-ribaudo isnt this SyntaxError? raw multiline strings are allowed only in template literals

Reply:
If I'm reading the JSX specification correctly, the only disallowed character is " (thus \n is allowed)
https://facebook.github.io/jsx/

Reply:
Indeed, it seems its allowed. Always thought that jsx is only a sugar for createElement calls and the rest is just JS. Dont like it doesnt conform to JS in those regards

Reply:
There is a lot of logic for strings in the transform at least